code/modules/plumbing/plumbers/bottler.dm

```diff
@@ -0,0 +1,79 @@
+/obj/machinery/plumbing/bottler
+	name = "chemical bottler"
+	desc = "Puts reagents into containers, like bottles and beakers."
+	icon_state = "bottler"
+	layer = ABOVE_ALL_MOB_LAYER
+	reagent_flags = TRANSPARENT | DRAINABLE
+	rcd_cost = 30
+	rcd_delay = 30
```

Comment:
these are worse than the synth stats, is that intended?

same for the fermentor.

Reply:
is 50 50 ok?